smart-contracts/assembly/contracts/website-deployer/websiteDeployer.ts

```diff
@@ -20,13 +23,19 @@ export function main(_: StaticArray<u8>): void {
 
   const websiteAddr = createSC(bytes);
 
-  // this will be updated when charging storage key for actual size
-  // const StorageKeyCreation = stringToBytes(OWNER_KEY).length * StorageCostPerByte;
+  const OwnerStorageKeyLen = stringToBytes(OWNER_KEY).length;
+  const OwnerStorageValueLen = stringToBytes(
+    Context.caller().toString(),
+  ).length;
+
+  const OwnerStorageCost =
+    baseCostBytes + OwnerStorageKeyLen + OwnerStorageValueLen;
+  const CreationTimestampStorageCost =
+    baseCostBytes + CreationTimestampKeyLen + CreationTimestampValueLen;
```

Comment:
it seems all this code will always computing the same value.
Imho, you should just put a constant with hardcoded value to save useless processing.
That said, i would suggest to keep this code commented here so we can understand where this value is from!